tests/test-recipes/split-packages/_intradependencies/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version='1' %}
+
+package:
+  name: intradependencies
+  version: {{ version }}
+
+requirements:
+  build:
+    - python {{ python }}
+    - r-base {{ r }}
+
+outputs:
+  - name: lib1intradependencies
+    script: install-lib1intradependencies.sh
+
+  - name: py1-intradependencies
+    requirements:
+      - {{ pin_subpackage('lib1intradependencies', exact=True) }}
+      - r-base {{ r }}
+    script: install-py1-intradependencies.sh
+
+  - name: r1-intradependencies
+    requirements:
+      - {{ pin_subpackage('lib1intradependencies', exact=True) }}
+      - python {{ python }}
+    script: install-r1-intradependencies.sh
+
+  - name: lib2intradependencies
```

Comment:
it's not clear to me what the relationship is between lib1 and lib2 here.  Can you please add a comment?  I think you're just making sure that lib1 stuff doesn't show up in lib2, right?